tsconfig.json

```diff
@@ -20,7 +20,13 @@
     "strictNullChecks": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+
+    /* Tests */
+    "baseUrl": "src",
```

Comment:
Come mai stiamo facendo degli alias? Così non rischiamo di confonderci? 
Sembra una lib esterna quando invece è una folder interna

Reply:
I don't think it creates confusion as the ctrl-click still brings the user to the local file.

It is also the React Testing library recommended setup: https://testing-library.com/docs/react-testing-library/setup/#configuring-jest-with-test-utils 

However, I'm open to other solutions, such as using @mia-platform-internal/testing-utils with a custom setup for this library.

Reply:
let's use the local path for now